<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveData</class>
 <widget class="QDialog" name="SaveData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>保存地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_lineEdit_file">
       <property name="text">
        <string>./data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btn_chose_file">
       <property name="text">
        <string>选择路径</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="m_btn_start">
       <property name="text">
        <string>开始保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btn_stop">
       <property name="text">
        <string>停止保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>保存帧数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_lineEdit_pfs_counts">
       <property name="text">
        <string>0</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_combo_model">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="m_lab_display_status">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
